src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Mappers/Apprentice/ViewApprenticehipUpdatesRequestToViewModelMapperTests.cs

```diff
@@ -0,0 +1,264 @@
+            public ApprenticeshipUpdate ApprenticeshipUpdate;
+
+            public long ApprenticeshipId = 1;
+
+            public ViewApprenticehipUpdatesRequestToViewModelMapperTestsFixture()
+            {
+                var autoFixture = new Fixture();
+                autoFixture.Customizations.Add(new DateTimeSpecimenBuilder());
+                CommitmentApiClient = new Mock<ICommitmentsApiClient>();
+
+                Source = new ViewApprenticehipUpdatesRequest { ApprenticeshipId = ApprenticeshipId, AccountId = 22, ApprenticeshipHashedId = "XXX", AccountHashedId = "YYY" };
+                GetApprenticeshipResponse = autoFixture.Create<GetApprenticeshipResponse>();
+                GetApprenticeshipResponse.Id = ApprenticeshipId;
+                autoFixture.RepeatCount = 1;
+                GetApprenticeshipUpdatesResponses = autoFixture.Create<GetApprenticeshipUpdatesResponse>();
+                ApprenticeshipUpdate = GetApprenticeshipUpdatesResponses.ApprenticeshipUpdates.First();
+                GetTrainingProgrammeResponse = autoFixture.Create<GetTrainingProgrammeResponse>();
+
+                var priceEpisode = new GetPriceEpisodesResponse
+                {
+                    PriceEpisodes = new List<GetPriceEpisodesResponse.PriceEpisode>(){ new GetPriceEpisodesResponse.PriceEpisode
+                {
+                    FromDate = DateTime.UtcNow.AddDays(-10),
+                    ToDate = null,
+                    Cost = 100
+                    } }
+                };
+
+                CommitmentApiClient.Setup(x => x.GetApprenticeship(ApprenticeshipId, It.IsAny<CancellationToken>())).Returns(() => Task.FromResult(GetApprenticeshipResponse));
+                CommitmentApiClient.Setup(x => x.GetApprenticeshipUpdates(ApprenticeshipId, It.IsAny<GetApprenticeshipUpdatesRequest>(), It.IsAny<CancellationToken>())).Returns(() => Task.FromResult(GetApprenticeshipUpdatesResponses));
+                CommitmentApiClient.Setup(x => x.GetPriceEpisodes(ApprenticeshipId, It.IsAny<CancellationToken>())).Returns(() => Task.FromResult(priceEpisode));
+                CommitmentApiClient.Setup(x => x.GetTrainingProgramme(It.IsAny<string>(), It.IsAny<CancellationToken>())).Returns(() => Task.FromResult(GetTrainingProgrammeResponse));
+
+                Mapper = new ViewApprenticehipUpdatesRequestToViewModelMapper(CommitmentApiClient.Object);
+            }
+
+            internal async Task<ViewApprenticeshipUpdatesRequestViewModel> Map()
+            {
+                return await Mapper.Map(Source) as ViewApprenticeshipUpdatesRequestViewModel;
+            }
+        }
+
+
+        public class DateTimeSpecimenBuilder : ISpecimenBuilder
+        {
+            public object Create(object request, ISpecimenContext context)
+            {
+                var pi = request as PropertyInfo;
+                if (pi == null || pi.PropertyType != typeof(DateTime?))
+                    return new NoSpecimen();
+
```

Comment:
could we include curly braces to this if statement as it is part of if/else

Reply:
done